indexing-service/src/main/java/org/apache/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -350,6 +351,7 @@ public void stop()
       return;
     }
     try {
+      log.info("Stopping...");
```

Comment:
I noticed the HttpRemoteTaskRunner and SegmentLoadDropHandler also just have logs that say `Starting...` and `Stopping...` To make the logs easier to follow, I think it's better to add some more identifying info to the logs.

`WorkTaskManager` also has a debug log `Starting...`

```suggestion
      log.info("Stopping RemoteTaskRunner...");
```

Would you mind fixing up these logs too if you're going to make another patch